addToCart/index.html

```diff
@@ -0,0 +1,28 @@
+<!<!DOCTYPE html>
+<html>
```

Comment:
in all documents let try to make the tabulation spacing of 2 and not 4

Reply:
Done, i have changed the preferences of my editor from 4 to 2 so from now i'm going to do it with 2 instead of 4 :)